app/services/stripev3.js

```diff
@@ -1 +1,8 @@
-export { default } from '@adopted-ember-addons/ember-stripe-elements/services/stripev3';
+import Service from "@adopted-ember-addons/ember-stripe-elements/services/stripev3";
```

Comment:
I don't believe so, since the existing service assumed a `config` object.
https://github.com/adopted-ember-addons/ember-stripe-elements/blob/master/addon/services/stripev3.js#L36

So this is a bug fix.